.github/workflows/test-build.yml

```diff
@@ -19,16 +13,30 @@ jobs:
       NX_CLOUD_ACCESS_TOKEN: ${{ secrets.NX_CLOUD_ACCESS_TOKEN }}
       NX_BRANCH: ${{ github.event.number || github.ref_name }}
     steps:
+      - name: Check for docs file changes
```

Comment:
Should we maybe split the jobs? The check and then the deploy that `needs` the check

Reply:
Nice one :+1: